### ISSUE TITLE
Task 5

### ISSUE THREAD

**Issue by jernejvivod** [jernejvivod/ny-parking-violations-analysis#10]
Perform the data analysis in a "batch" manner using machine learning to predict events such as
days with a high number of tickets (think of and implement at least one additional learning problem).
You will need to appropriately transform the augmented data.
Ensure that the workers will not have enough memory to store and process the entire dataset
(e.g., 8GB per worker). Use at least three kinds of supervised machine learning algorithms:
- One of the simple distributed algorithms from Dask-ML
- a complex third-party algorithm which "natively" supports distributed computing (XGBoost, LightGBM, ...)
- One of the common scikit-learn algorithms utilizing partial\_fit.

For all these scenarios, compare performance in terms of loss (error), scalability, time, and total
memory consumption.
